src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier fs = new FigureSupplier();
+        Figure [] randomFigures = {fs.getRandomFigure(),
+                fs.getRandomFigure(), fs.getRandomFigure()};
+        Figure [] defaultFigures = {fs.getDefaultFigure(),
+                fs.getDefaultFigure(), fs.getDefaultFigure()};
```

Comment:
random and default figures can be in one array
also we can fill array by using cycle